app/src/main/java/org/schabi/newpipe/App.java

```diff
@@ -80,7 +82,9 @@ protected void attachBaseContext(Context base) {
     @Override
     public void onCreate() {
         super.onCreate();
+        this.context = this;
 
+        ThemeHelper.init(context);
```

Comment:
Why did you create a member variable that you only reference once? Couldn't you just do `ThemeHelper.init(this)`? It would be more readable, I think

Reply:
Done